src/py42/modules/detectionlists.py

```diff
@@ -123,3 +123,16 @@ def remove_user_cloud_alias(self, user_id, alias):
         factory = self._microservice_client_factory
         self._detection_list_user_client = factory.get_detection_list_user_client()
         self._detection_list_user_client.remove_cloud_alias(user_id, alias)
+
+    def refresh_user_scim_attributes(self, user_id):
```

Comment:
Could you add a link to the relevant REST documentation that is behind this method to the docstring?

Reply:
Added.